t/unit/utils/test_time.py

```diff
@@ -253,3 +254,41 @@ def test_utcoffset(self, patching):
         assert utcoffset(time=_time) is not None
         _time.daylight = False
         assert utcoffset(time=_time) is not None
+
+
+class test_get_exponential_backoff_interval:
+
+    @patch('random.randrange', lambda n: n - 2)
+    @patch('random.choice', lambda sequence: sequence[0])
```

Comment:
Sorry this is actually unused. Could you please remove it? Or I can do it if you allow me changes.